frontend/src/index.tsx

```diff
@@ -1,14 +1,32 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
+import { Auth0Provider } from './react-auth0-spa';
 import App from './components/app';
+import config from './auth_config.json';
+import history from './utils/browserHistory';
 import { store } from './app/store';
 import * as serviceWorker from './serviceWorker';
 
+// Routes user to correct url after authenticating
+const onRedirectCallback = (appState: any) => {
```

Comment:
In Typescript it's a good idea to avoid `any` type as much as possible. My guess would be we'll need to make an interface for appState, but does it work if we take `appState: any` out completely?

Reply:
Okay good to know - originally it had no type there but that didn't work. I'll create an interface and get that sorted.

Reply:
I've updated this to use an interface. It works (I've tested redirecting somewhere and it lands you at the page you want).